auth0/v3/authentication/passwordless.py

```diff
@@ -31,34 +31,58 @@ def email(self, client_id, email, send='link', auth_params=None):
             send (str, optional): Can be: 'link' or 'code'. Defaults to 'link'.
 
             auth_params (dict, optional): Parameters to append or override.
+
+            client_secret (str): Client Secret of the application.
         """
 
         return self.post(
             'https://{}/passwordless/start'.format(self.domain),
             data={
                 'client_id': client_id,
+                'client_secret': client_secret,
                 'connection': 'email',
                 'email': email,
                 'send': send,
                 'authParams': auth_params
             }
         )
 
-    def sms(self, client_id, phone_number):
-        """Start flow sending a SMS message.
+    def sms(self, client_id, phone_number, client_secret=None):
+        """Start flow sending an SMS message.
+
+        Given the user phone number, it will send an SMS with 
+        a verification code. You can then authenticate with
+        this user using phone number as username and code as password.
+
+        Args:
+            client_id (str): Client Id of the application.
+
+            client_secret (str): Client Secret of the application.
+
+            phone_number (str): Phone number.
         """
 
         return self.post(
             'https://{}/passwordless/start'.format(self.domain),
             data={
                 'client_id': client_id,
+                'client_secret': client_secret,
```

Comment:
Does this ensure we don't send the arg if the value is not provided?

Reply:
Solved